guides/common/modules/proc_creating-an-installation-medium-for-debian.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Creating_an_Installation_Medium_for_Debian_{context}"]
+= Creating an Installation Medium for Debian
+
+Create an installation medium in {Project} to provision hosts running Debian or Ubuntu.
+
+ifdef::orcharhino[]
+[TIP]
+====
+You can use an Ansible role to configure operating systems to skip this step.
+The Ansible role is located on your orcharhino Server at `/usr/share/orcharhino-ansible/roles/or_operating_systems/`.
+====
+endif::[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > Installation Media*.
+. Click *Create Medium* to create an installation medium entry.
+. Enter a *Name* for the installation medium.
+. Set the *Path* to the upstream URL `\http://ftp.debian.org/debian/`.
++
+Alternatively, you can also use a Debian Mirror.
+Note that synchronized content from {Project} does not work for two reasons: first, the `linux` and `initrd.gz` files are not synchronized; second, the `Release` file is not signed with the official Debian GPG private key.
```

Comment:
You could be more specific on what exactly does not work. Perhaps instead stating that something does not work, which generally is something that RHD does not do, we could tell what needs to be done in order to provision the host. I would assume if files were `wget/curled` on the TFTP server it could work. Anyway, not blocking.